src/gui/gt_icons.h

```diff
@@ -61,7 +130,7 @@ GT_GUI_EXPORT QIcon colorize(const QIcon& icon, SvgColorData colorData);
  *  @param getDisabledColor Color function pointer for a disabled icon.
  * @return Icon
  */
-GT_GUI_EXPORT QIcon colorize(const QIcon& icon,
+GT_GUI_EXPORT QIcon colorize(const gt::gui::Icon& icon,
```

Comment:
I'd use `Icon` here as well, as you did everywhere else in this file

```suggestion
GT_GUI_EXPORT QIcon colorize(const Icon& icon,
```